ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -106,33 +106,20 @@ tests:
     cluster_profile: packet
     env:
       ASSISTED_CONFIG: |
-        IPv6=yes
-        IPv4=no
-        VIP_DHCP_ALLOCATION=no
-        PROXY=yes
-        OPENSHIFT_VERSION=4.8
+        IPv6=true
+        TEST_FUNC=test_install
+        MAKEFILE_TARGET="create_full_environment run test_parallel"
     workflow: baremetalds-assisted-install
 - as: e2e-metal-assisted-ipv6-periodic
   cron: '@daily'
-  steps:
-    cluster_profile: packet
-    env:
-      ASSISTED_CONFIG: |
-        IPv6=yes
-        IPv4=no
-        VIP_DHCP_ALLOCATION=no
-        PROXY=yes
-    workflow: baremetalds-assisted
```

Comment:
Can you please write the reason this job was removed in the commit message?

Reply:
It was a temporary job. But I agree it should be mentioned in the commit message @eliorerz

Reply:
Sure